fhttp/httprunner.go

```diff
@@ -94,7 +94,8 @@ func RunHTTPTest(o *HTTPRunnerOptions) (*HTTPRunnerResults, error) {
 		aborter:     r.Options().Stop,
 	}
 	httpstate := make([]HTTPRunnerResults, numThreads)
-	warmup := errgroup{}
```

Comment:
@howardjohn I am a bit puzzled on how moving the declaration of that variable changes the behavior?
`warmup` isn't yet used, no function on it is called... so weird side effect?

Reply:
moving the variable isn't the fix, that was just to keep it closer to its usage. The real fix is doing two loops, so we initialize all HTTPClients in serial and then start the calls in parallel